patches/openwrt/0008-ipq40xx-add-support-for-the-ZyXEL-NBG6617.patch

```diff
@@ -80,7 +80,7 @@ to get the challange value/seed.
 |012345678901
 
 This seed/value can be converted to the password with the help of this
-bash script (Thanks to http://www.adslayuda.com/Zyxel650-9.html authors):
+bash script (Thanks to https://www.adslayuda.com/zyxel650-actualizacion_del_firmware.html authors):
```

Comment:
you can't edit this one, this is a commit in openwrt that we simply cherry-picked.

Reply:
Just thought that the old link might become invalid in the future. I will remove the commit.

Reply:
a commit message written by someone else, contained in a public repository, is written into stone for eternity ;-) we are simply copying. also, this copy will become obsolete as soon as Gluon is based on OpenWrt 19.x